modules/core/src/test/java/org/apache/ignite/internal/suggestions/JvmConfigurationSuggestionsTest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
```

Comment:
I know that most people don't case but technically this should be 2021. I don't know why they put year into a license header, that was a horrible idea

Reply:
Fixed the year